torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -847,9 +847,6 @@ def analyze_kernel_mutations(
             # (e.g. `tt.elementwise_inline_asm`), we assume it does not mutate any input parameters.
             if op.name in UNKNOWN_OPS:
                 if op.name == "tt.elementwise_inline_asm" and op.is_pure:
-                    log.warning(
```

Comment:
Would it still be useful as a debug log?

Reply:
IMO it doesn't really matter, debug log or no log are both fine